erpnext/selling/sales_common.js

```diff
@@ -97,7 +97,7 @@ erpnext.selling.SellingController = erpnext.TransactionController.extend({
 	customer: function() {
 		var me = this;
 		erpnext.utils.get_party_details(this.frm, null, null,
-			function(){ me.apply_pricing_rule() });
+			function(){ me.apply_pricing_rule(), me.apply_price_list()});
```

Comment:
You cannot call 2 async functions back-to-back.

apply_pricing_rule should return a promise.

see who `frappe.run_serially` is used.

it should be 

```
return frappe.run_serially([
  me.apply_pricing_rule(),
  me.apply_price_list()
])
```

Reply:
update: @nabinhait says `apply_pricing_rule` is already called within `apply_price_list`

Please check